multitenancy-spring-boot-autoconfigure/src/main/java/org/axonframework/extensions/multitenancy/autoconfig/MultiTenancyAxonServerAutoConfiguration.java

```diff
@@ -233,6 +244,7 @@ public EventProcessorInfoConfiguration processorInfoConfiguration(
                     c.getComponent(AxonServerConfiguration.class)
             );
             tenantProvider.subscribe(controlService);
+            controlService.start();
```

Comment:
Is this start necessary? The `EventProcessorInfoConfiguration` is a `ModuleConfiguration`. AF will grab all `ModuleConfiguration` instances and invoke the `ModuleConfiguration#initialize(Configuration)` method on them.
In the case of the `EventProcessorInfoConfiguration`, the implementation is the following:

```java
@Override
public void initialize(Configuration config) {
    this.config = config;
    // if there are no event handlers registered, there may be no EventProcessingConfiguration at all.
    if (config.eventProcessingConfiguration() != null) {
        this.config.onStart(Phase.INBOUND_EVENT_CONNECTORS, eventProcessorControlService::get);
    }
}
```

The `get` invocation of the `eventProcessorControlService` variable is retrieving the `MultiTenantEventProcessorControlService` from a `Component`. 
Due to this, its lifecycle methods **should** be invoked.

Hence why I am asking whether this is necessary. :-)
Was this change a precaution or necessity?